packages/inventory-compliance/src/RulesComplianceFilter.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { FilterDropdown } from '@redhat-cloud-services/frontend-components';
+
+const FILTER_CATEGORIES = [
+    {
+        type: 'checkbox', title: 'Passed', urlParam: 'passed', values: [
+            { label: 'True', value: 'passed' },
+            { label: 'False', value: 'failed' }
+        ]
+    },
+    {
+        type: 'checkbox', title: 'Severity', urlParam: 'severity', values: [
```

Comment:
Can we have the icons in this? It serves as a sort-of legend to have them
![image](https://user-images.githubusercontent.com/23172038/58733511-b5e3a580-83c2-11e9-89d1-bf2c5bc8dd1d.png)

Reply:
Apparently the whole thing doesn't render if I try to display icons - https://github.com/RedHatInsights/frontend-components/pull/38 is meant to fix it

Reply:
Also that "Low" squared icon is nowhere to be found on https://patternfly-react.surge.sh/patternfly-4/icons/Icons/ :disappointed_relieved: